spec/decorators/notification_decorator_spec.rb

```diff
@@ -74,4 +74,119 @@
       expect(notification.decorate.milestone_count).to eq("64")
     end
   end
+
+  describe "reaction to a article" do
```

Comment:
I made a judgement call here to not test some of the `siblings` / `actors` logic in the decorator. I don't feel great about that - I'd like to have more tests - but it's a lot to setup in tests, that setup is pretty brittle (easy to get false confidence, much harder to get right) and - in my opinion - should be part of a larger refactor, later on.